### PR TITLE
[BALDE-1] Window

### PR DESCRIPTION
**Pull request by Blinkuu**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -3,4 +3,4 @@ project(bald_engine)
 
 set(CMAKE_CXX_STANDARD 17)
 
-add_library(bald_engine library.cpp library.h)
\ No newline at end of file
+add_library(bald_engine library.cpp library.h graphics/window.h)
```
Comment (piotrmoszkowicz) [inline CMakeLists.txt]: Wtf?

File: graphics/window.h [hunk 2]
```diff
@@ -0,0 +1,12 @@
+//
+// Created by Lukasz Gut on 14/12/2018.
+//
+
+#pragma once
+
+#include <iostream>
+
+class Window {
```
Comment (piotrmoszkowicz) [inline graphics/window.h]: Ftw

**Review by piotrmoszkowicz — CHANGES_REQUESTED** [review]
No nie no

File: CMakeLists.txt (hunk 1 above)
Comment (BIGbadEL) [reply CMakeLists.txt]: xDD

Comment (4jeR) [reply CMakeLists.txt]: O

**Review by piotrmoszkowicz — APPROVED** [review]

**Pull request closed without merging**